chef/cookbooks/cpe_chrome/resources/cpe_chrome.rb

```diff
@@ -94,5 +96,31 @@ def manage_chrome_osx
         ],
       }
     end
+
+    unless node['cpe_chrome']['mp']['UseMasterPreferences'] == false
+      mprefs = node['cpe_chrome']['mp']['FileContents'].reject { |v| v.nil? }
```

Comment:
Will test, but we want to reject the variable entirely, not just remove nil values. I think there is some confusion as to what is trying to be achieved here.